packages/hub/node-tests/routes/email-card-drop-verify-test.ts

```diff
@@ -86,6 +93,19 @@ describe('GET /email-card-drop/verify', function () {
     expect(response.headers['location']).to.equal(`${webClientUrl}${success}`);
   });
 
+  it('rejects a used email hash', async function () {
+    await emailCardDropRequestsQueries.insert(unclaimedEoaWithClaimedEmailHash);
+
+    let response = await request().get(
+      `/email-card-drop/verify?eoa=${unclaimedEoaWithClaimedEmailHash.ownerAddress}&verification-code=${unclaimedEoaWithClaimedEmailHash.verificationCode}&email-hash=${unclaimedEoaWithClaimedEmailHash.emailHash}`
+    );
+
+    expect(provisionPrepaidCardCalls).to.equal(0);
+
+    expect(response.status).to.equal(400);
+    expect(response.text).to.equal('Email has already been used to claim a prepaid card');
```

Comment:
Should Card Wallet be updated to handle this known error?

Reply:
Card Wallet will not know about this error, because this route is visited in the browser.

Reply:
oh yes of course hehe thx 🙃